### ISSUE TITLE
Some of the start/end dates came up as null

### ISSUE THREAD

**Issue by KiprasJ** [bigfoott/ScrapedDuck#3]
Hello,

I am noticing some missing start/end dates. Example:
```
[
   {
      "eventID":"season-12-master-league_evolution-cup",
      "name":"Master League and Evolution Cup | Season of Light",
      "eventType":"go-battle-league",
      "heading":"GO Battle League",
      "link":"https://leekduck.com/events/season-12-master-league_evolution-cup/",
      "image":"https://leekduck.com/assets/img/events/gobattleleague-master-key.jpg",
      "start":null,
      "end":"2022-10-13T20:00:00.000Z",
      "extraData":null
   },
   {
      "eventID":"mega-manectric-in-mega-raids-october-2022",
      "name":"Mega Manectric in Mega Raids",
      "eventType":"raid-battles",
      "heading":"Raid Battles",
      "link":"https://leekduck.com/events/mega-manectric-in-mega-raids-october-2022/",
      "image":"https://leekduck.com/assets/img/events/mega-default.jpg",
      "start":null,
      "end":"2022-10-20T10:00:00.000",
      "extraData":{
         "raidbattles":{
            "bosses":[
               {
                  "name":"Mega Manectric",
                  "image":"https://leekduck.com/assets/img/pokemon_icons/pokemon_icon_310_51.png",
                  "canBeShiny":true
               }
            ],
            "shinies":[
               
            ]
         }
      }
   },
   {
      "eventID":"shiny-corsola-okinawa-shirt-pikachu",
      "name":"Pikachu wearing an Okinawan kariyushi shirt and Shiny Corsola",
      "eventType":"location-specific",
      "heading":"Location-specific",
      "link":"https://leekduck.com/events/shiny-corsola-okinawa-shirt-pikachu/",
      "image":"https://leekduck.com/assets/img/events/pikachu-okinawa.jpg",
      "start":null,
      "end":null,
      "extraData":null
   }
]
```

**Comment by bigfoott** [comment]
Hi! Sorry about the delay, I completely missed this.

This appears to be an issue with how these specific events are shown on the LeekDuck page.

How the events page works is there are two event items on the page, one in the current section and one in the upcoming section. Then based on the local time, one of the two events is hidden. I'm not sure why these events are missing their start dates unfortunately, and without those extra elements on the page itself I don't believe it'd be possible to get this info.

I'll be reaching out to Leek to see if there's a specific edge case for these I've missed, and how best to handle it :)

Also a side note, the Shiny corsola event is null because we haven't been given an end date for that event lol

**Comment by KiprasJ** [comment]
Thanks for the response, yeah I already dug into the code as well to see where the issue might be. But it seems it depends on the elements which are on the actual event page from LeekDuck. Anyway, a nice feature. I'm using your JSON for a discord bot, but its more for personal use. Anyways, I'm glad that I found this because I started writing my own version of a similar scraper. Keep up with a good work! 😄

**Comment by bigfoott** [comment]
Np! I'm glad you've found this useful! 😄 I'll let you know when I hear back from Leek and/or I implement some sort of workaround.

**Comment by KiprasJ** [comment]
Seems like there are more issues coming in:
```
{
        "eventID": "november-communityday2022",
        "name": "Teddiursa Community Day",
        "eventType": "community-day",
        "heading": "Community Day",
        "link": "https://leekduck.com/events/november-communityday2022/",
        "image": "https://leekduck.com/assets/img/events/cd-2022-11-teddiursa.jpg",
        "start": "2022-11-12T14:00:00.000",
        "end": "2022-11-12T17:00:00.000",
        "extraData": {
            "communityday": {
                "communityday": {
                    "communityday": {
                        "communityday": {
                            "communityday": {
                                "communityday": {
                                    "communityday": {
                                        "communityday": {
                                            "communityday": {
                                                "communityday": {
                                                    "communityday": {
                                                        "communityday": {
                                                            "communityday": {
                                                                "communityday": {
                                                                    "communityday": {
                                                                        "communityday": {
                                                                            "communityday": null
                                                                        }
                                                                    }
                                                                }
                                                            }
                                                        }
                                                    }
                                                }
                                            }
                                        }
                                    }
                                }
                            }
                        }
                    }
                }
            }
        }
    },
```

**Comment by bigfoott** [comment]
Hey! Wow, that's a new one lol. I've fixed the issue causing this in cd18489307c8ac4b98fbee6e32d634b8a7525e95 and 5f9a02ecfe08b03e386873e43aeb6032559a3f54.

**Comment by bigfoott** [comment]
Hi! Sorry for the delay on replying to this issue. This was actually fixed in c8b331a42368387d33a77ad41a980390b948db20 last week, I forgot to close this issue though.

The date data is now pulled from the events feed provided by Leek, so both values should always be populated now, unless there's an issue on the LeekDuck end :)
